### DISCUSSION TITLE
something bugs fixed and i how to update the frame that include all package?

### DISCUSSION THREAD

**Discussion by lkzwc** [facebook/docusaurus#7527] [Q&A]
all package

**Comment by Josh-Cena** [comment]
Please see https://docusaurus.io/docs/installation#updating-your-docusaurus-version

**Reply by lkzwc** [reply]
![image](https://user-images.githubusercontent.com/84896877/170877362-14c04d56-f6e8-451e-a868-91e5fe409187.png)
and others?

**Reply by Josh-Cena** [reply]
You don't need to update others.

**Reply by lkzwc** [reply]
ok thanks
